bellatrix.playwright/src/main/java/solutions/bellatrix/playwright/components/common/create/RelativeCreateService.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 Automate The Planet Ltd.
+ * Author: Miriam Kyoseva
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package solutions.bellatrix.playwright.components.common.create;
+
+import solutions.bellatrix.core.plugins.EventListener;
+import solutions.bellatrix.core.utilities.InstanceFactory;
+import solutions.bellatrix.playwright.components.WebComponent;
+import solutions.bellatrix.playwright.components.common.ComponentActionEventArgs;
+import solutions.bellatrix.playwright.components.contracts.Component;
+import solutions.bellatrix.playwright.findstrategies.FindStrategy;
+import solutions.bellatrix.playwright.services.ComponentCreateService;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RelativeCreateService extends ComponentCreateService {
```

Comment:
Not sure what is the ide behind RelativeCreateService - it looks like a regular CreateComponenService

Reply:
As of now, in the bellatrix.web module, when you create components through other components, the syntax is different.
```
Div divComponent = app().create().byId(Div.class, "outer-div");
Div anotherDivComponent = divComponent.createById(Div.class, "inner-div");
```

For the playwright module, I made this RelativeCreateService, that will allow creating components through other components with the same syntax:
```
Div divComponent = app().create().byId(Div.class, "outer-div");
Div anotherDivComponent = divComponent.create().byId(Div.class, "inner-div");
```

The difference between RelativeCreateService and ComponentCreateService is in the methods by() and allBy(), which are the ones called inside methods like byXpath().
In RelativeCreateService, the native playwright method for finding an element through another element is called,
while in ComponentCreateService, the method for finding elements through the Page object is called.

The other way is made available for Playwright module, to lessen the refactoring in case a project is migrated to use the Playwright engine.